spec/takeaway_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'takeaway'
+
+describe Takeaway do
+  # let(:menu) { double :menu, price: '£1.00', contains?: true }
+  # subject(:order) { described_class.new(menu) }
+  context 'starting up with printable menu' do
+    it "should initialize with a hash menu" do
+      expect(subject.menu.dishes).to be_a Hash
+    end
+
+    it "should print dishes on show_menu" do
+      expect { subject.show_menu }.to output.to_stdout
+    end
+  end
+
+  context 'ordering food' do
+    it 'should only accept items from the menu' do
```

Comment:
Could use double instead of hard coding.

`let(:invalid_choice) { double("invalid_choice") }`

However, you won't really be changing the  `invalid_choice`, so maybe hard coding here is fine.